students/philip_behrend/lesson03/assignment/basic_operations.py

```diff
@@ -0,0 +1,49 @@
+""" Operations file to interact with Customer database """
+import logging
+import traceback
+from customer_model import *
+
+logging.basicConfig(level=(logging.INFO))
+logger = logging.getLogger(__name__)
+
+def add_customer(**kwargs):
+    """ Add customer function """
+    try:
+        with db.transaction():
+            new_cust = (Customer.create)(**kwargs)
+            new_cust.save()
+            logger.info('Customer addition successful')
+    except Exception as e:
```

Comment:
Avoid using catch all exceptions. They're ok for debug code (when you simply need things to pass for a very restricted set of test cases), but the pose a risk on the long run, in that new issues not observed during the development process are inadvertently, an incorrectly, handled by the general exception. Replace it instead with a specific exception(s) that you know can occur or remove the try/except block altogether.